package.json

```diff
@@ -340,6 +340,7 @@
     "electron-builder": "^22.7.0",
     "electron-notarize": "^0.3.0",
     "eslint": "^7.7.0",
+    "execa": "^4.1.0",
```

Comment:
Not needed anymore?

Reply:
No, good catch (though it might be useful in the future since it makes working with `child_process` a lot easier